lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1796,15 +1790,10 @@ emitAssociatedFunctionReference(ADContext &context, SILBuilder &builder,
         context.lookUpMinimalDifferentiationTask(originalFn, desiredIndices);
     if (!task) {
       if (originalFn->isExternalDeclaration()) {
-        // For LLDB REPL, we should attempt to load the function as
-        // this may be defined in a different cell.
-        if (isInLLDBREPL(*original->getModule()))
-          original->getModule()->loadFunction(originalFn);
-        // If we still don't have the definition, generate an error message.
-        if (originalFn->isExternalDeclaration()) {
+        if (!originalFn->isDefinition()) {
```

Comment:
Note: the [old condition](https://github.com/apple/swift/commit/01339e7832297d00d7124d1e1ea0cf89aa8dde99#diff-4ede198777b59748c377cdd17c444e83L1653) here seems only to check for `if (originalFn->isExternalDeclaration())`.

Reply:
Aren't they equivalent? In any case,  I will change this to the old condition to be consistent with the rest of the codebase.

Reply:
Done.